ops.json

```diff
@@ -23,12 +23,6 @@
     "level": 4,
     "bypassesPlayerLimit": true
   },
-  {
-    "uuid": "e82603df-417d-4a95-bd1f-1720648be0b4",
-    "name": "Pablete1234",
-    "level": 4,
-    "bypassesPlayerLimit": true
-  },
```

Comment:
ayo??

Reply:
[:P](https://github.com/bolt-rip/config-pgm/pull/40/files#diff-9a02247b768deb1e2fe20f55c60c6e3ac5ad3e0843ddd0251a41c153a06c4e11L21-L25)

Reply:
And i'm not allowed to be special and be in there twice unlike everyone else? >:(